R/draws.R

```diff
@@ -1,10 +1,112 @@
+#' This method can be adopted by specifying `method` through [method_condmean()] with
+#' argument `type = "jackknife"`.
+#'
+#' The argument `data` must have one row per visit per subject. This means that incomplete
+#' outcome data must be set as `NA` instead of having the related row missing. Missing values
+#' in the covariates are not allowed. If `data` is incomplete
+#' then the [expand_locf()] helper function can be used to add in any missing rows, using
+#' Last Observation Carried Forward (LOCF) imputation to impute the covariates values.
+#'
+#' The argument `data_ice` contains information about the occurrence of ICEs. It is a
+#' `data.frame` with 3 columns:
+#' - A vector of characters related to the ids of the subjects that experienced
+#'   the ICE (must be named as `vars$subjid`).
+#' - A vector of levels of `data[[vars$visit]]` including the first visit after the ICE (must be named as `vars$visit`). If
+#'   multiple ICEs happen for the same subject, then only the first non-MAR visit should be set.
+#' - A vector of characters related to the imputation strategy to address the ICE (must be named as `vars$strategy`).
+#'   Possible imputation strategies are:
+#'   - `"MAR"`: Missing At Random.
+#'   - `"JTR"`: Jump To Reference.
+#'   - `"CIR"`: Copy Increments in Reference.
+#'   - `"CR"`: Copy Reference.
+#'   - `"JR"`: Jump to Reference.
+#'   - `"LMCF"`: Last Mean Carried Forward.
+#'   - User-defined imputation strategies can also be set.
+#'
+#' This argument is necessary at this stage since the model will be fit only on those observed data
+#' that, if missing, would have been characterized as MAR. Thus, any non-MAR observed data, i.e.
+#' any observed data after `data_ice[[vars$visit]]` that is addressed with an imputation
+#' strategy different from MAR will be discarded for the model fit. However such observed data
+#' will not be discarded from the data in the imputation phase
+#' (performed with the function ([impute()]). To summarize, **at this stage only pre-ICE data
+#' and MAR post-ICE data are used**.
+#'
+#' The argument `vars` is a named list of key variable names to specify
+#' the imputation model. Can be set using [set_vars()] providing the following arguments:
+#' - `subjid`: column in `data` related to the subject ids variable.
+#' - `visit`: column in `data` related to the visit variable.
+#' - `outcome`: column in `data` related to the outcome variable.
+#' - `group`: column in `data` related to the group variable.
+#' - `covariates`: vector of characters related to the covariates to be included
+#'   in the model (including interactions).
+#'   By default the model specification is `outcome ~ 1 + visit + group`.
+#'   Please note that the `group*visit` interaction
+#'   is **not** included in the model by default.
+#' - `strata`: covariates used as stratification variables in the bootstrap.
+#'   By default only the `vars$group` is used as stratification variable.
```

Comment:
Potentially pedantic but I would pefer `set to`  instead of `used` as it might imply that we use group regardless of what the user specifies here.

Reply:
done